example/build_cmake_installed/CMakeLists.txt

```diff
@@ -10,8 +10,10 @@ project(Example CXX Fortran)
 find_package(Kokkos REQUIRED)
 
 add_executable(example cmake_example.cpp foo.f)
+MESSAGE(STATUS "XXX COMPILER_ID: ${CMAKE_Fortran_COMPILER_ID}")
```

Comment:
In case that helps, it looks like the `LLVMFlang` was only added in CMake 3.24 https://cmake.org/cmake/help/v3.24/variable/CMAKE_LANG_COMPILER_ID.html